ecloud/resource_instance.go

```diff
@@ -78,6 +81,11 @@ func resourceInstance() *schema.Resource {
 				Type:     schema.TypeBool,
 				Optional: true,
 			},
+			"data_volume_ids": {
```

Comment:
Wondering whether it'd be better to invert the attachment of volumes here - rather than specifying against instance, we could specify against the volume instead, or even have a volume_attachment resource? Not sure what is best

Reply:
I had originally had  it that way with the volume resources referencing an instance ID, but it felt a bit clunky doing that.  
Especially during instance delete, as instance delete currently deletes all attached volumes (data and os volumes), so declaring instance attachments against the volume resource isn't really authoritative at the moment.